swerve100/src/main/java/frc/robot/subsystems/SwerveModuleFactory.java

```diff
@@ -48,15 +48,15 @@ public static SwerveModule WCPModule(
 
         // DRIVE FF
         SimpleMotorFeedforward driveFeedforward = new SimpleMotorFeedforward( //
-                0.05, // kS: from experiment; overcome friction for low-effort moves
-                .5, // kV
-                0.1); // kA: I have no idea what this value should be
+                0.07, // kS: from experiment; overcome friction for low-effort moves
+                .15);// kV
```

Comment:
i'm surprised that zero kA could be made to work without overshoot.   what happened with nonzero kA?  maybe instead you could smooth out the acceleration estimate?